### PR TITLE
Pass textmode variable to the power_action 

### PR DESCRIPTION
**Pull request by b10n1k**
The Desktop variable is different than the expected. use of textmode as desktop which will trigger the reboot in power_action

- Related ticket: https://progress.opensuse.org/issues/52418
- Verification run: http://aquarius.suse.cz/tests/694#
http://aquarius.suse.cz/tests/695

### CODE REVIEW

File: tests/console/validate_lvm_raid1.pm [hunk 1]
```diff
@@ -36,8 +36,12 @@ sub run {
     _check_raid1_partitioning($config, $actual_num_devs);
     _remove_raid_disk($config, $actual_num_devs);
     _reboot();
-    $self->wait_boot;
-
+    if (is_opensuse) {
+        select_console('root-console');
```
Comment (jknphy) [inline tests/console/validate_lvm_raid1.pm]: you will be selecting `root-console` twice, here and in `_check_raid_disks_after_reboot`.

**Comment by jknphy** [comment]
Job is setting in lower case `DESKTOP` setting https://openqa.opensuse.org/tests/1040608, so the job is ignoring it and selecting `kde` instead https://openqa.opensuse.org/tests/1040608/file/vars.json. I see you use that in your verification. Could you adjust `DESKTOP=minimalx`in O3 as well?
We also need to unschedule `opensuse_welcome` test module.
It looks to me like [grub](https://openqa.opensuse.org/tests/1028862#step/validate_lvm_raid1/56) is not always caught.
Was this done by the automation? http://aquarius.suse.cz/tests/680#step/validate_lvm_raid1/64

File: schedule/yast/lvm_raid1/lvm+raid1_opensuse.yaml [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
+  - installation / resolve_dependency_issues
+  - installation / installation_overview
+  - installation / disable_grub_timeout
+  - installation / start_install
+  - installation / await_install
+  - installation / logs_from_installation_system
+  - installation / reboot_after_installation
+  - installation / grub_test
+  - installation / first_boot
+  - console / validate_lvm_raid1
```
Comment (jknphy) [inline schedule/yast/lvm_raid1/lvm+raid1_opensuse.yaml]: Check this file, something strange happen with spaces.

File: tests/console/validate_lvm_raid1.pm [hunk 3]
```diff
@@ -17,7 +17,7 @@ use testapi;
 use utils;
 use Test::Assert ':all';
 use scheduler 'get_test_data';
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_opensuse);
```
Comment (jknphy) [inline tests/console/validate_lvm_raid1.pm]: you don't need now `is_opensuse`

**Comment by jknphy** [comment]
btw you can also use `git commit --amend` to adapt description of your commit.

File: tests/console/validate_lvm_raid1.pm (hunk 3 above)
Comment (b10n1k) [reply tests/console/validate_lvm_raid1.pm]: ee1b3981a resolved

File: schedule/yast/lvm_raid1/lvm+raid1_opensuse.yaml (hunk 2 above)
Comment (b10n1k) [reply schedule/yast/lvm_raid1/lvm+raid1_opensuse.yaml]: ee1b3981a resolved

**Comment by jknphy** [comment]
These are the missing points not addressed yet:
 - Add VR for SLE to probe that not only works for openSUSE.
 - (optional) ammend the commit.

**Comment by b10n1k** [comment]
http://aquarius.suse.cz/tests/695

**Comment by jknphy** [comment]
LGTM

**Pull request merged**